java/integTest/src/test/java/glide/PubSubTests.java

```diff
@@ -1468,4 +1468,59 @@ public void pubsub_channels_and_numpat_and_numsub_in_transaction(boolean standal
                 },
                 result);
     }
+
+    @SneakyThrows
+    @Test
+    public void pubsub_shard_channels() {
+        assumeTrue(SERVER_VERSION.isGreaterThanOrEqualTo("7.0.0"), "This feature added in version 7");
+
+        // no channels exists yet
+        GlideClusterClient client = (GlideClusterClient) createClient(false);
+        assertEquals(0, client.pubsubChannels().get().length);
+        assertEquals(0, client.pubsubChannelsBinary().get().length);
+        assertEquals(0, client.pubsubChannels("**").get().length);
```

Comment:
"**" is not defined anywhere. Can you include a comment what you're doing here?

Reply:
Borrowed from [PubSubTests#pubsub_channels()](https://github.com/valkey-io/valkey-glide/blob/main/java/integTest/src/test/java/glide/PubSubTests.java#L1264), but
1. it's just a matching pattern, probably should use `*` instead of `**`;
2. should use `pubsubShardChannels` instead of `pubsubChannels`, as we are testing it.

Will fix these 2